app/services/proforma_service/cache_import_file.rb

```diff
@@ -25,7 +25,7 @@ def execute
     private
 
     def file_data_hash(task, import_file, proforma_task)
-      {path: proforma_task[:path].tr(' ', '_'),
+      {path: proforma_task[:path],
```

Comment:
There is also `Filenam#sanitized`, which may be more useful. But hard to tell without domain knowledge.